library/Fission/Internal/Mock/Effect/Types.hs

```diff
@@ -143,3 +145,7 @@ data DestroyLoosePin
   = DestroyLoosePin     UserId IPFS.CID
   | DestroyLoosePinById UserId LoosePinId
   deriving (Eq, Show)
+
+data APICall 
+  = APICall
```

Comment:
Let's perhaps be clearer about this one. `FissionWebRequest` or the more general `WebRequest` if it can also do that. Thoughts?

Reply:
Actually... we got rid of the mock that used this so I'm just going to get rid of the Effect type as well